### PR TITLE
Files from the tarball src

### PR DESCRIPTION
**Pull request by mrmdrogers**
Hey @mattunlv ,

It looks to me like we may want to go through the changes and make sure they are good before merging this branch in to master.

You can make comments on a line by line basis if you see things you want to keep from master, and I will update the PR.

### CODE REVIEW

File: src/AST/AltStat.java [hunk 1]
```diff
@@ -5,49 +5,20 @@
 public class AltStat extends Statement {
```
Comment (mattunlv) [inline src/AST/AltStat.java:5]: Keep this version of AltStat as it has the replicated alt features to be used in future developments - note, nothing in the current version uses the replicated alt features.

File: src/AST/ArrayType.java [hunk 2]
```diff
@@ -5,34 +5,16 @@
 public class ArrayType extends Type {
```
Comment (mattunlv) [inline src/AST/ArrayType.java:5]: Keep this version.

File: src/AST/ExternType.java [hunk 3]
```diff
@@ -25,8 +25,4 @@ public String signature() {
     public <S extends Object> S visit(Visitor<S> v) {
```
Comment (mattunlv) [inline src/AST/ExternType.java:25]: keep this version - was added when external types were added.

File: src/AST/NamedType.java [hunk 4]
```diff
@@ -16,8 +16,6 @@ public NamedType(Name name) {
     public NamedType(Name name, Type type) {
```
Comment (mattunlv) [inline src/AST/NamedType.java:16]: keep this file - this change was probably done when external types were added.

File: src/CodeGeneratorJava/CodeGeneratorJava.java [hunk 5]
```diff
@@ -73,15 +73,6 @@
 import AST.UnaryPreExpr;
```
Comment (mattunlv) [inline src/CodeGeneratorJava/CodeGeneratorJava.java:73]: KEEP THIS ONE ;-)

File: src/CodeGeneratorJava/Helper.java [hunk 6]
```diff
@@ -93,25 +93,27 @@ static boolean isYieldingProc(ProcTypeDecl pd) {
      * Returns the wrapper class name of primitive data types.
```
Comment (mattunlv) [inline src/CodeGeneratorJava/Helper.java:93]: keep this file

File: src/Parser/ProcessJ.cup [hunk 7]
```diff
@@ -1212,7 +1206,6 @@ alt_body ::=
 alt_case ::=
           LPAREN expression:e RPAREN ANDAND guard:g COLON statement:s {: RESULT = new AltCase(e, g, s); :}
         | guard:g COLON statement:s                                   {: RESULT = new AltCase(null, g, s); :}
-        | alt_statement
```
Comment (mattunlv) [inline src/Parser/ProcessJ.cup:1215]: This line must be removed for now!

File: src/ProcessJ/runtime/TimerQueue.java [hunk 8]
```diff
@@ -42,14 +41,13 @@ public void run() {
...
-                        synchronized (p) { 
+                        synchronized (p) { // ADDED
                             p.setReady();
-                        } 
+                        } // ADDED
                     }
-                    size--;
                 }
             } catch (InterruptedException e) {
-//                System.err.println("[TimerQueue] Unexpected interrupt exception encountered.");
```
Comment (mattunlv) [inline src/ProcessJ/runtime/TimerQueue.java:52]: remove //

File: src/Utilities/Log.java [hunk 9]
```diff
@@ -3,7 +3,7 @@
 import AST.AST;
 
 public class Log {
-    public static boolean doLog = true;
```
Comment (mattunlv) [inline src/Utilities/Log.java:6]: set back to false ;-) and allow to alter ..

**Review by mattunlv — CHANGES_REQUESTED** [review]

**Comment by mrmdrogers** [comment]
Closing this PR because the tarball was out of date.

The changes are now in https://github.com/mattunlv/ProcessJ/pull/23

**Pull request closed without merging**